spec/features_spec.rb

```diff
@@ -1,11 +1,42 @@
 # (c) 2017 Ribose Inc.
 #
 
+# No point in using ApplicationRecord here.
+# rubocop:disable Rails/ApplicationRecord
+
 require "spec_helper"
 
 RSpec.describe "Attr Masker gem" do
-  example "The Rake task doesn't raise any exceptions" do
-    expect { run_rake_task }.not_to raise_exception
+  before do
+    stub_const "User", Class.new(ActiveRecord::Base)
+  end
+
+  let!(:han) do
+    User.create!(
+      first_name: "Han",
+      last_name: "Solo",
+      email: "han@example.test",
+    )
+  end
+
+  let!(:luke) do
+    User.create!(
+      first_name: "Luke",
+      last_name: "Skywalker",
+      email: "luke@jedi.example.test",
+    )
+  end
+
+  example "Masking a single text attribute with default options" do
+    User.class_eval do
+      attr_masker :last_name
+    end
+
+    expect { run_rake_task }.not_to(change { User.count })
+
+    expect { han.reload }.to(change { han.last_name }.to("(redacted)"))
```

Comment:
I agree with @skalee that `.not_to change{ }` is more readable and seems more like the rspec style. 

I don't have a strong preference between `expect { }` and `expect{}` since they are as readable, but if in the first case the `change` block will need to be put in parenthesis. @abunashir what do you think?